src/js/actions/GuideActions.js

```diff
@@ -14,7 +14,8 @@ module.exports = {
   },
 
   retrieveGuidesToFollow: function (election_id) {
-    Dispatcher.loadEndpoint("voterGuidesToFollowRetrieve", { google_civic_election_id: election_id });
+    Dispatcher.loadEndpoint("voterGuidesToFollowRetrieve", { google_civic_election_id: election_id,
```

Comment:
Isn't it clearer to have more than 1 property on separate lines? This all gets minified afterwards.